butler-bot/bot.js

```diff
@@ -63,4 +68,46 @@ client.on('message', message => {
     .then(() => { logger.info('Successfully added the role') })
 })
 
+client.setInterval((servers) => {
+  const itMuslimServer = servers.get(IT_MUSLIM_SERVER_ID)
+
+  const incompleteMembers = itMuslimServer.members.filter((member) => {
+    return !member.roles.has(FULL_MEMBER_ROLE_ID)
+  })
+  /* console.log(incompleteMembers) */
+
+  const incompleteMembersName = incompleteMembers.map((item) => {
+    return item.user.username
+  })
+  /* console.log(incompleteMembersName) */
+
+  const membersJoinTime = incompleteMembers.map((member) => {
```

Comment:
If it's a time period, then call it `time`, if it's a point in time, call it `timestamp`:
```suggestion
  const memberJoinTimestamps = incompleteMembers.map((member) => {
```